hazelcast/src/main/java/com/hazelcast/cardinality/impl/hyperloglog/HyperLogLog.java

```diff
@@ -30,7 +31,7 @@
  * </ul>
  *
  */
-public interface HyperLogLog extends IdentifiedDataSerializable {
+public interface HyperLogLog extends IdentifiedDataSerializable, Versioned {
```

Comment:
I think we need this to be `Versioned` to super cluster version checks in the IDS methods.